src/app/view/components/dot-relationship-tree/dot-relationship-tree.component.scss

```diff
@@ -0,0 +1,114 @@
+@use "variables" as *;
+
+:host {
+    ul {
+        list-style: none;
+        padding-left: $spacing-5;
+        li {
+            span {
+                margin-left: $spacing-1;
+            }
+            .dot-tree__content {
+                display: flex;
+                align-items: center;
+            }
+        }
+    }
+    ul.dot-tree {
+        list-style: none;
+        padding: 0;
+        margin: 0;
+        position: relative;
+        & > li::after {
+            background: $gray-light;
+            width: 1px;
+            height: 19px;
+            content: "•";
+            position: absolute;
+            top: 19px;
+            left: 11px;
+            color: $gray-light;
+            display: flex;
+            justify-content: space-evenly;
+            align-items: end;
+            font-size: $font-size-x-large;
+            line-height: 1px;
+        }
+    }
+    ul.dot-tree--child {
+        position: relative;
+        & > li::after {
```

Comment:
Break line after each class.

Does anybody know if we can do this with prettier?